### ISSUE TITLE
Failed to run BaseObfuscator - Web UI Not Loading

### ISSUE THREAD

**Issue by wilczekj** [mitre/caldera#2943]
**Describe the bug**
When deploying an operation that makes use of the steganography Obfuscator, the web UI becomes unresponsive and I cannot log back in without a refresh. This has happened for me now on both 4.2.0 and 5.0.0. 

**To Reproduce**
1. Create an operation
2. Choose steganography for the Obfuscator
3. Deploy

**Expected behavior**
For steganography to work. If there is an underlying package or config needed, there should be some error or documentation.

**Screenshots**
Image shows the UI spinning and not loading after trying to use Steganography Obfuscator.
![image](https://github.com/mitre/caldera/assets/88460965/5071b810-3e88-4882-9843-05a58fc70f64)

**Desktop (please complete the following information):**
 - OS: Caldera 4.2.0 on Ubuntu 22.04
 - Browser:  Chrome Version 123.0.6312.106 (Official Build) (64-bit) and Edge Version 123.0.2420.81 (Official build) (64-bit)
 - Version: 4.2.0

**Additional context**
Error logs when the UI is unresponsive.
```
2024-04-10 19:14:37 - ERROR (base_obfuscator.py:21 run) Failed to run BaseObfuscator, returning default decoded bytes
2024-04-10 19:14:37 - DEBUG (connectionpool.py:978 _new_conn) Starting new HTTPS connection (1): aws.random.cat:443
2024-04-10 19:16:48 - ERROR (base_obfuscator.py:21 run) Failed to run BaseObfuscator, returning default decoded bytes
2024-04-10 19:16:48 - DEBUG (connectionpool.py:978 _new_conn) Starting new HTTPS connection (1): aws.random.cat:443
2024-04-10 19:18:59 - ERROR (base_obfuscator.py:21 run) Failed to run BaseObfuscator, returning default decoded bytes
2024-04-10 19:18:59 - DEBUG (connectionpool.py:978 _new_conn) Starting new HTTPS connection (1): aws.random.cat:443
2024-04-10 19:21:13 - ERROR (base_obfuscator.py:21 run) Failed to run BaseObfuscator, returning default decoded bytes
2024-04-10 19:21:13 - DEBUG (connectionpool.py:978 _new_conn) Starting new HTTPS connection (1): aws.random.cat:443
```

Running a tcpdump shows traffic flowing to port 8888, but caldera is unresponsive.

**Comment by wilczekj** [comment]
It happens with any Operation using any Adversary. Are there any additional logs I can look at.

**Comment by wilczekj** [comment]
Are you on Ubuntu 22.04?

I performed an install exactly per the docs for version 5.0.0 and then experienced a ton of issues (some of which are currently tracked as issues, others were not)
Example issues on 5.0.0 (both of which were solved using the suggested steps)
- https://github.com/mitre/caldera/issues/2911
- https://github.com/mitre/caldera/issues/2883 

After all the issues with 5.0.0, I created a new Ubuntu 22.04 VM and installed 4.2.0 where I encountered many of the same issues.

In both instances, this was a base Ubuntu install with no other packages/installations installed beyond those requested in the Caldera install instructions (which are very, very minimal - meaning it's 4 commands, copy/pasted to the command line - not sure how I would have messed that up twice).
